app/index.js

```diff
@@ -88,8 +91,24 @@ App.prototype = {
 
 
   listen: function(port, env) {
-    this.koaApp.listen(port);
-    console.log('Application started:', { port: port, env: env });
+    var httpPort = parseInt(port);
+    var httpsPort = httpPort + 10000;
+
+    http.createServer(this.koaApp.callback()).listen(httpPort);
+    console.log('Application started:', { port: httpPort, env: env });
+
+    if (process.env.SERVE_HTTPS === 'true') {
```

Comment:
Could you refactor this code to a separate method like "_listenHTTPS" or something like that?